### ISSUE TITLE
NVDA doesn't read spell check in Microsoft Office Word 2010 correctly

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3714]
**Reported by The_Dark_Man on 2013-12-12 15:53**
When I start the spell check in Word 2010, NVDA doesn't read the next misstake, sometimes. Sometimes, it works correct. I must push the escape key and F7 to continue the correction. That's time-consuming. Jaws has the same problem.

**Comment by leonardder** [comment]
This is related to #3410, although that's a newer version in which Microsoft has changed a lot, according to @JCSTeh. However, since it is also related to usability issues regarding to spell checking, closing this in favour of #3410. May be we can discuss all the spell checker related issues there, regardless of Word version.
